website/source/docs/job-specification/volume.html.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: "docs"
+page_title: "volume Stanza - Job Specification"
+sidebar_current: "docs-job-specification-volume"
+description: |-
+   The "volume" stanza allows the group to specify that it requires a given volume
+   from the cluster. Nomad will automatically handle ensuring that the volume is
+   available, and mounted into the task.
+---
+
+# `volume` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> group -> **volume**</code>
+    </td>
+  </tr>
+</table>
+
+The "volume" stanza allows the group to specify that it requires a given volume
+from the cluster. Nomad will automatically handle ensuring that the volume is
```

Comment:
based on the docs, I'm not sure how it would ensure something is available (e.g. creating a resource) - does this mean it ensure that a client exposes the said volume rather than creating the resource?